code/handlers/trackservers.q

```diff
@@ -175,12 +173,15 @@ retrydiscovery:{
 // Called by the discovery service when it restarts
 autodiscovery:{if[DISCOVERYRETRY>0; .servers.retrydiscovery[]]}
 
+//check if .proc.getattributes exists in the nontorqprocess
+checkNonTorq:{$[(`proc in key`)&(first distinct`getattributes`cd in key`.proc);1;0]};
```

Comment:
no other function here appears to use camelCase, so for consistency (and because it's awful) please avoid